sdk/identity/Azure.Identity/tests/CredentialTestHelpers.cs

```diff
@@ -0,0 +1,346 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Microsoft.Identity.Client;
+
+namespace Azure.Identity.Tests
+{
+    internal class CredentialTestHelpers
```

Comment:
Utility code from various credentials live tests is extracted into this class so that it can be reused it `DefaultAzureCredentialLiveTests`